app-catalog/src/components/releases/List.tsx

```diff
@@ -8,10 +8,22 @@ import {
 } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
 import { Box } from '@mui/material';
 import { useEffect, useState } from 'react';
+import { fetchLatestAppVersion } from '../../api/charts';
 import { listReleases } from '../../api/releases';
 
+const formatVersion = (v?: string) => {
+  const s = (v ?? '').trim();
+
+  if (!s || s === '—') {
+    return '—';
+  }
+
+  return s.startsWith('v') ? s : `v${s}`;
```

Comment:
I don't think we should add a v to it if it's not there. Some version numbers don't use v in there, so adding it could be confusing to people.